File: js/cliente.js
const form = document.querySelector(".form");

form.addEventListener("submit", async (e) => {
    e.preventDefault();

    const name = document.querySelector("#name").value;
    const surname = document.querySelector("#surname").value;
    const cpf = document.querySelector("#cpf").value;
    const email = document.querySelector("#email").value;
    const birthday = document.querySelector("#birthday").value;

    const cliente = {
        nome: name,
        sobrenome: surname,
        cpf: cpf,
        email: email,
        dataNascimento: birthday
    }
    
    Post(cliente);
});

async function Post(cliente) {
    console.log(cliente);

    const url = "http://localhost:5013/cliente";
    const request = await fetch(url, {
        method: "POST",
        headers: {
            "Content-Type": "application/json"
        },
        body: JSON.stringify(cliente)
        
    });

    if(!request.ok) {
        console.log("Deu erro");
    }

    const response = await request.json();

    console.log(response);  

    alert("Dados enviados");
}

File: backend/Models/Cliente.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

public class Cliente 
{
    [Required]
    [Key]
    public string? Cpf { get; set; }

    [Required]
    public string? Nome { get; set; }
    [Required]
    public string? Sobrenome { get; set; }
    [Required]
    public string? Email { get; set; }
    [Required]
    public DateOnly DataNascimento { get; set; }

    
}